src/configs/battery.rs

```diff
@@ -4,11 +4,11 @@ use starship_module_config_derive::ModuleConfig;
 
 #[derive(Clone, ModuleConfig)]
 pub struct BatteryConfig<'a> {
-    pub full_symbol: &'a str,
-    pub charging_symbol: &'a str,
-    pub discharging_symbol: &'a str,
-    pub unknown_symbol: &'a str,
-    pub empty_symbol: &'a str,
+    pub full_symbol: Option<&'a str>,
```

Comment:
No, this will make users overwrite the default config with None, even if one does not specify the key `full_symbol`. Same updates should be done with charging and discharging symbol.

Use this.

```suggestion
    pub full_symbol: &'a str
```

Reply:
Ok, sorry you are right. I will update it.